package.json

```diff
@@ -3,17 +3,17 @@
   "description": "Explore the Merkle Forest from the comfort of your browser",
   "version": "1.1.0",
   "private": true,
+  "type": "module",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build --stats",
-    "test": "react-scripts test --env=node",
-    "test:coverage": "react-scripts test --coverage",
-    "test:serve": "ecstatic build --cache=0 --port=3001",
-    "test:e2e": "react-scripts test --roots test/ --testMatch '**/*.test.js'",
-    "test:e2e:ci": "URL=http://localhost:3001 run-p --race test:serve test:e2e",
-    "analyze": "webpack-bundle-analyzer build/bundle-stats.json",
+    "start": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "test": "run-s test:unit test:e2e",
+    "test:unit": "vitest run --environment=node",
+    "test:unit-playwright": "playwright test -c playwright-ct.config.js",
```

Comment:
not currently used, but available to run unit tests via playwright if we want to normalize on tests.